common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -148,14 +149,19 @@ public static ByteBuffer getIncreasingIntBuffer(int len) {
   }
 
   /**
-   * * Writes buffer to the given file path
+   * Writes buffer to the given file path. Overwrites this file if exists.
```

Comment:
I think technically, this method deletes the file first, if it already exists, not overwriting it.